core-bundle/config/services.yaml

```diff
@@ -1056,6 +1056,13 @@ services:
         calls:
             - [setName, [contao_backend]]
 
+    contao.session.contao_backend_popup:
+        class: Contao\CoreBundle\Session\Attribute\ArrayAttributeBag
+        arguments:
+            - _contao_be_popup_attributes
+        calls:
+            - [setName, [contao_backend]]
+
```

Comment:
It does not make sense to define the session bags as services if the session factory does not provide a way to dynamically add bags. For this reason, I would prefer to simplify the setup like I did in https://github.com/contao/contao/pull/7269/files#diff-c61014971921decbcd3006c93d1ab6ac2ba12fd11cdc69699cd597d361316117.